app/assets/javascripts/select-kit/addon/components/color-palette-picker/color-palette-picker-row.gjs

```diff
@@ -0,0 +1,28 @@
+import { computed } from "@ember/object";
+import { classNames } from "@ember-decorators/component";
+import ColorPalettePreview from "discourse/components/color-palette-preview";
+import SelectKitRowComponent from "select-kit/components/select-kit/select-kit-row";
+
+@classNames("color-palette-picker-row")
+export default class ColorPalettePickerRow extends SelectKitRowComponent {
+  @computed("item.id", "value")
+  get active() {
+    return this.item?.id === this.value;
+  }
+
+  <template>
+    {{#if this.active}}
+      <span class="color-palette-picker-row__badge --active">
+        {{! todo: use i18n (where should i put it?) }}
```

Comment:
We have `user.color_schemes` on line 1548 in client.en.yml. What would you say about `user.color_schemes.picker.active` ?

Reply:
Sounds good!